addon/-private/system/model/internal-model.js

```diff
@@ -636,7 +632,7 @@ export default class InternalModel {
       changedKeys = this._changedKeys(data.attributes);
     }
 
-    emberAssign(this._data, data.attributes);
+    Object.assign(this._data, data.attributes);
```

Comment:
Can't rely on `Object.assign`. Using `Ember.assign` instead (IE11 does not have `Object.assign`)...